### PR TITLE
Update StyleCop

### PR DESCRIPTION
**Pull request by philipktlin**
Update StyleCop version and fix style errors.

### CODE REVIEW

File: Microsoft.Azure.Devices.Edge.sln [hunk 1]
```diff
@@ -562,24 +562,24 @@ Global
 		{A68ABCD4-5926-4005-9525-559DD74991C9}.Debug|Any CPU.Build.0 = Debug|Any CPU
 		{A68ABCD4-5926-4005-9525-559DD74991C9}.Release|Any CPU.ActiveCfg = Release|Any CPU
 		{A68ABCD4-5926-4005-9525-559DD74991C9}.Release|Any CPU.Build.0 = Release|Any CPU
-		{950DACB0-B011-41AF-B0FB-245F749B01AB}.CheckInBuild|Any CPU.ActiveCfg = Debug|Any CPU
-		{950DACB0-B011-41AF-B0FB-245F749B01AB}.CheckInBuild|Any CPU.Build.0 = Debug|Any CPU
+		{950DACB0-B011-41AF-B0FB-245F749B01AB}.CheckInBuild|Any CPU.ActiveCfg = CheckInBuild|Any CPU
+		{950DACB0-B011-41AF-B0FB-245F749B01AB}.CheckInBuild|Any CPU.Build.0 = CheckInBuild|Any CPU
 		{950DACB0-B011-41AF-B0FB-245F749B01AB}.CodeCoverage|Any CPU.ActiveCfg = Debug|Any CPU
```
Comment (darobs) [inline Microsoft.Azure.Devices.Edge.sln]: I know this fixes the CheckInBuild, but should some of these CodeCoverage builds get fixed as well?

File: edge-agent/test/Microsoft.Azure.Devices.Edge.Agent.Core.Test/commands/LoggingCommandFactoryTest.cs [hunk 2]
```diff
@@ -11,8 +11,8 @@ namespace Microsoft.Azure.Devices.Edge.Agent.Core.Test.Commands
     using Microsoft.Extensions.Logging;
     using Moq;
     using Xunit;
-    using CommandMethodExpr = System.Linq.Expressions.Expression<System.Func<ICommandFactory, System.Threading.Tasks.Task<ICommand>>>;
-    using TestExecutionExpr = System.Func<LoggingCommandFactory, System.Threading.Tasks.Task<ICommand>>;
+    using CommandMethodExpr = System.Linq.Expressions.Expression<System.Func<Microsoft.Azure.Devices.Edge.Agent.Core.ICommandFactory, System.Threading.Tasks.Task<Microsoft.Azure.Devices.Edge.Agent.Core.ICommand>>>;
```
Comment (darobs) [inline edge-agent/test/Microsoft.Azure.Devices.Edge.Agent.Core.Test/commands/LoggingCommandFactoryTest.cs]: I think I'd prefer a using statement and then avoid "Microsoft.Azure.Devices.Edge.Agent.Core" here.

File: Microsoft.Azure.Devices.Edge.sln (hunk 1 above)
Comment (philipktlin) [reply Microsoft.Azure.Devices.Edge.sln]: CodeCoverage is not used anymore; I will clean this configuration later.

File: edge-agent/test/Microsoft.Azure.Devices.Edge.Agent.Core.Test/commands/LoggingCommandFactoryTest.cs (hunk 2 above)
Comment (philipktlin) [reply edge-agent/test/Microsoft.Azure.Devices.Edge.Agent.Core.Test/commands/LoggingCommandFactoryTest.cs]: for type used in using statement; it doesn't reference to other using statement.  Therefore with stylecop checking, i need to provide it as full qualified name.

**Pull request merged**